src/main/java/de/tum/in/www1/artemis/web/rest/lecture/VideoUnitResource.java

```diff
@@ -0,0 +1,139 @@
+package de.tum.in.www1.artemis.web.rest.lecture;
+
+import static de.tum.in.www1.artemis.web.rest.util.ResponseUtil.*;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import de.tum.in.www1.artemis.domain.Lecture;
+import de.tum.in.www1.artemis.domain.lecture.VideoUnit;
+import de.tum.in.www1.artemis.repository.LectureRepository;
+import de.tum.in.www1.artemis.repository.VideoUnitRepository;
+import de.tum.in.www1.artemis.service.AuthorizationCheckService;
+import de.tum.in.www1.artemis.web.rest.util.HeaderUtil;
+
+@RestController
+@RequestMapping("/api")
+public class VideoUnitResource {
+
+    @Value("${jhipster.clientApp.name}")
+    private String applicationName;
+
+    private final Logger log = LoggerFactory.getLogger(ExerciseUnitResource.class);
+
+    private static final String ENTITY_NAME = "videoUnit";
+
+    private final VideoUnitRepository videoUnitRepository;
+
+    private final LectureRepository lectureRepository;
+
+    private final AuthorizationCheckService authorizationCheckService;
+
+    public VideoUnitResource(LectureRepository lectureRepository, AuthorizationCheckService authorizationCheckService, VideoUnitRepository videoUnitRepository) {
+        this.lectureRepository = lectureRepository;
+        this.authorizationCheckService = authorizationCheckService;
+        this.videoUnitRepository = videoUnitRepository;
+    }
+
+    /**
+     * GET /video-units/:videoUnitId : gets the video unit with the specified id
+     *
+     * @param videoUnitId the id of the videoUnit to retrieve
+     * @return the ResponseEntity with status 200 (OK) and with body the video unit, or with status 404 (Not Found)
+     */
+    @GetMapping("/video-units/{videoUnitId}")
```

Comment:
```suggestion
    @GetMapping("/lectures/{lectureId}/video-units/{videoUnitId}")
```

Reply:
Let's stay consistent with the API

Reply:
I did it this way because I tried to follow the guidelines: 
![image](https://user-images.githubusercontent.com/29718932/97961964-835a8b00-1db4-11eb-894b-797a40a9ccf0.png)
Should I change it for all units?

Reply:
I do not fully agree to the marked statement and would prefer, that our REST URLs stay consistent of starting points

Reply:
so yes, please change it ;-)

Reply:
Done in 
2781c727467c1856877faaeda668c20accef330c
9d7f2cd06db2e196187a6dc14ea4986d9e218472
72bab48290d7f14c4d8fd31d641e52676521dc18